test/integration/harness.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+# This test harness is needed to support NetBSD, whose env fails to find python3
+# by its usual name (though this works everywhere else), so we cannot rely on
+# shebang to work properly and ask meson to solve the problem for us.
+
+test="$1"
+[ -f "$test" ] || exit 1
+exec "$PYTHON_PATH" "$@"
```

Comment:
Ugh, that's nasty. Maybe there is a better way, but let's keep it like this then and perhaps fix it later if possible.

Reply:
Hmm, it's not clear to me why you need this. You can run a test in meson as

```meson
test('testname', python, args: target)
```

Instead of using harness.sh as the test program and passing the python script as an argument, use python as the test program and pass the python script as an argument.

You're still not relying on the shebang working properly on NetBSD, indeed.

Reply:
Doesn't work there. Add shebang and `env` can't find Python. Remove shebang and it starts failing with "exec format error". I can't remember now if I tried to remove executable flags from files, I believe I did, and it also didn't work. Will try again soon.